indexing-service/src/main/java/org/apache/druid/indexing/common/actions/RetrieveUsedSegmentsAction.java

```diff
@@ -58,17 +62,22 @@ public class RetrieveUsedSegmentsAction implements TaskAction<Collection<DataSeg
   @JsonIgnore
   private final Segments visibility;
 
+  @JsonIgnore
+  private final boolean onlyLocked;
+
   @JsonCreator
   public RetrieveUsedSegmentsAction(
       @JsonProperty("dataSource") String dataSource,
       @Deprecated @JsonProperty("interval") Interval interval,
       @JsonProperty("intervals") Collection<Interval> intervals,
       // When JSON object is deserialized, this parameter is optional for backward compatibility.
       // Otherwise, it shouldn't be considered optional.
-      @JsonProperty("visibility") @Nullable Segments visibility
+      @JsonProperty("visibility") @Nullable Segments visibility,
+      @JsonProperty("onlyLocked") @Nullable Boolean onlyLocked
```

Comment:
Rather than adding a new boolean property, we should try to incorporate this into the definition of the `Segments` enum.

Reply:
This change is no longer being made as part of this PR